test/fuzzer/pedro/nested_subquery_table_function.test

```diff
@@ -1,12 +0,0 @@
-# name: test/fuzzer/pedro/nested_subquery_table_function.test
-# description: Issue #4676: Subquery as Table UDF argument assertion error
-# group: [pedro]
-
-statement ok
```

Comment:
Instead of removing this test, can we adapt it to the new results?

Reply:
I included this test in mine ([here](https://github.com/DBatUTuebingen/duckdb/blob/8e9b1517d596c1f6964db052418bec37812516b7/test/sql/table_function/range_correlated.test#L98)), but I can also keep it in its original location and use a table function that still fails (which one?).

Reply:
Ah if you've only moved them then that's fine, I was just worried about reducing the coverage, disregard this comment 👍